### ISSUE TITLE
Please redesign the dialog

### ISSUE THREAD

**Issue by pcav** [lequynhhoa/Check-Defined-Convert-CRS#3]
It does not shrink correctly, labels are cut in some systems.
Thanks.

**Comment by lequynhhoa** [comment]
My dialog is inheritanced in my tools (Check coordinate system and Convert coordinate system) that I made last time

**Comment by pcav** [comment]
Yes, but still it does not follow minimal Human Interface Giudelines.
Thanks for checking.

**Comment by lequynhhoa** [comment]
Yes. I will check it later. Thank you
